autoscheduler/frontend/src/components/NavBar.tsx

```diff
@@ -23,6 +23,23 @@ const useStyles = makeStyles((theme) => ({
 const NavBar: React.SFC = () => {
   const classes = useStyles(appTheme);
 
+  const [usersName, setUsersName] = React.useState('');
+
+  function getUsersName(): void {
+    fetch('sessions/get_full_name').then(
+      (res) => res.json(),
+    ).then((result) => {
+      if (result.full_name) {
+        setUsersName(result.full_name);
+      }
+    });
+    if (usersName === '') {
+      setUsersName('Google Login');
+    }
+  }
+
+  getUsersName();
```

Comment:
You can change the initial value of `usersName` to `Google Login` and instead of calling `getUsersName` directly, use `React.useEffect` to make this a little cleaner (also notice how I destructure the json to simplify it a little bit):
```suggestion
  const [usersName, setUsersName] = React.useState('Google Login');
  React.useEffect(() => {
    fetch('sessions/get_full_name').then(
      (res) => res.json(),
    ).then(({ full_name }) => {
      if (full_name) setUsersName(full_name};
    });
  }, []);
```